### PR TITLE
Peer identification extension

### PR DESCRIPTION
**Pull request by KanishkT123**
Support for peer identification extension, including unit tests and updated readme.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const PeerList = require('webrtc-signal-http/lib/peer-list')
 const signalRouterCreator = require('webrtc-signal-http')
 const heartbeatRouterCreator = require('webrtc-signal-http-heartbeat')
 const capacityRouterCreator = require('webrtc-signal-http-capacity')
+const recognitionRouterCreator = require('webrtc-signal-http-peer-identification/lib')
```
Comment (bengreenier) [inline lib/index.js]: we should change `package.json#main` to avoid the `/lib` here

File: lib/index.js [hunk 2]
```diff
@@ -100,6 +101,20 @@ module.exports = (opts) => {
...
+            peerList: peerList
+        })
+
+        // use the recognition peer list instead
+        peerList = recognitionRouter.peerList
+
+        app.use(recognitionRouter)
+    }
+
+
```
Comment (bengreenier) [inline lib/index.js]: nit: extra lines

File: test/basic.js [hunk 3]
```diff
@@ -71,6 +71,39 @@ describe('3dtoolkit-signal', () => {
...
+            assert.equal(app.peerList.dataFor(clientId2).split("\n").length, 4)
+            assert.equal(app.peerList.dataFor(clientId3).split("\n").length, 4)
+
+            //Make sure these are the same lists being returned 
+            assert.equal(app.peerList.dataFor(clientId1), app.peerList.dataFor(clientId2))
+            assert.equal(app.peerList.dataFor(clientId1), app.peerList.dataFor(clientId3))
+            
+            //Make sure there are no clients in the returned peer list
+            assert(!app.peerList.dataFor(clientId1).includes("client"))
+
```
Comment (bengreenier) [inline test/basic.js]: nit: extra line

File: lib/index.js (hunk 1 above)
Comment (KanishkT123) [reply lib/index.js]: Fixed!

**Pull request merged**